config/jvm.options

```diff
@@ -59,11 +56,7 @@
 #-XX:HeapDumpPath=${LOGSTASH_HOME}/heapdump.hprof
 
 ## GC logging
-#-Xlog:gc*,gc+age=trace,safepoint:file=@loggc@:utctime,pid,tags:filecount=32,filesize=64m
-
-# log GC status to a file with time stamps
-# ensure the directory exists
-#-Xloggc:${LS_GC_LOG_FILE}
+#-Xlog:gc*,gc+age=trace,safepoint:file=${LS_GC_LOG_FILE}:utctime,pid,tags:filecount=32,filesize=64m
```

Comment:
is Xloggc no longer available? otherwise I'd keep it as someone may have changes only Xloggc (and not Xlog)

Reply:
with  `-Xloggc` JVM prints the deprecation error:
```
[0.001s][warning][gc] -Xloggc is deprecated. Will use -Xlog:gc:/tmp/gclog instead.
```

after manual replace of `${LS_GC_LOG_FILE}`with `/tmp/gclog`